vars/retryWithSleep.groovy

```diff
@@ -32,6 +32,7 @@ def call(Map args = [:], Closure body) {
   def seconds = args.get('seconds', 10)
   def backoff = args.get('backoff', false)
   def sleepFirst = args.get('sleepFirst', false)
+  def sideEffect = args.get('sideEffect', { })
```

Comment:
The word sideEffect usually has a negative connotation. What do you think about using `callback` instead?

Reply:
If that was your intention, I'm OK too :)

Reply:
very tiny neat:

For those shared library steps that support closures, we use the signature `def call (Map args=[:], Closure body)` so maybe we could use the same class type for clarity?

```suggestion
  def sideEffect = args.get('sideEffect', Closure)
```